lib/ansible/modules/network/nxos/nxos_vxlan_vtep_vni.py

```diff
@@ -302,6 +315,7 @@ def main():
     if module.params['assoc_vrf']:
         mutually_exclusive_params = ['multicast_group',
                                      'suppress_arp',
+                                     'suppress_arp_disable',
```

Comment:
@saichint I wonder why this check is even needed. `mutually_exclusive` can be just passed to AnsibleModule.
```
mutually_exclusive = [('suppress_arp', 'suppress_arp_disable'),
                      ('assoc_vrf', 'multicast_group', 'suppress_arp', ...)]
```

Reply:
@trishnaguha Thanks for pointing this out. I also thought of it but the code was like this from the beginning and I did not want rewrite it but you are right, it was unnecessary. Please note that, the parameters like multicast_group, suppress_arp etc. are mutually exclusive only with assoc_vrf, not among themselves. The only exception is suppress_arp and suppress_arp_disable. I made the changes accordingly.

Reply:
Thanks.